src/components/Calculator.js

```diff
@@ -0,0 +1,473 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+
+/* eslint-disable no-eval */
+function Calculator() {
+  const [data, setData] = useState('');
```

Comment:
- [ ] Your app is not rendering anything on the server, Please make sure your app runs fine and matches with the given design.

### See the image of the error.

![image](https://user-images.githubusercontent.com/83803414/164460569-da30cb7f-928f-46f3-b2fa-bc56601e8fc8.png)